pkg/kubectl/cmd/proxy.go

```diff
@@ -35,21 +35,10 @@ import (
 var (
 	defaultPort = 8001
 	proxyLong   = templates.LongDesc(i18n.T(`
-		To proxy all of the kubernetes api and nothing else, use:
-
-		    $ kubectl proxy --api-prefix=/
-
-		To proxy only part of the kubernetes api and also some static files:
-
-		    $ kubectl proxy --www=/my/files --www-prefix=/static/ --api-prefix=/api/
```

Comment:
can you move these to the Examples section rather than deleting it. I think once you remove them here, we won't have any other documentation explaining these. It's ok for Examples section to be long, we can clean that up later.